test/framework/type_checking.py

```diff
@@ -293,10 +294,10 @@ def test_to_dependency(self):
 
         # no name/version
         self.assertErrorRegex(EasyBuildError, "Can not parse dependency without name and version: .*",
-            to_dependency, {'toolchain': 'lib, 1.2.8', 'versionsuffix': 'suff'})
+                              to_dependency, {'toolchain': 'lib, 1.2.8', 'versionsuffix': 'suff'})
         # too many values
-        self.assertErrorRegex(EasyBuildError, "Found unexpected \(key, value\) pair: .*",
-            to_dependency, {'lib': '1.2.8', 'foo':'1.3', 'toolchain': 'lib, 1.2.8', 'versionsuffix': 'suff'})
+        dep_spec = {'lib': '1.2.8', 'foo': '1.3', 'toolchain': 'lib, 1.2.8', 'versionsuffix': 'suff'}
+        self.assertErrorRegex(EasyBuildError, r"Found unexpected \(key, value\) pair: .*", to_dependency, dep_spec)
```

Comment:
There is a subtle difference here, the original does not have r"Found... but only "Found...
How much of a difference does this result in in reality?

Reply:
The `r"..."` only makes it a raw string, which is necessary to shut up the Hound w.r.t. the escaped `(` and `)` characters.

This check verifies whether the produced exception matches the specified pattern, so if the test passes (which it does), all is good.

Reply:
ok, just checking.